src/lib/components/admin/Settings/Models/Manage/ManageOllama.svelte

```
@@ -0,0 +1,1062 @@
+									/>
+									<path
+										d="M3.5 9.75a.75.75 0 0 0-1.5 0v1.5A2.75 2.75 0 0 0 4.75 14h6.5A2.75 2.75 0 0 0 14 11.25v-1.5a.75.75 0 0 0-1.5 0v1.5c0 .69-.56 1.25-1.25 1.25h-6.5c-.69 0-1.25-.56-1.25-1.25v-1.5Z"
+									/>
+								</svg>
+							</button>
+						</div>
+					</div>
+
+					{#if createModelDigest !== ''}
+						<div class="flex flex-col mt-1">
+							<div class="font-medium mb-1">{createModelTag}</div>
```

ESLint

[error] 'createModelTag' is not defined.

(no-undef)